### PR TITLE
Home

### PR DESCRIPTION
**Pull request by HelaGdh**

### CODE REVIEW

**Comment by NasriHaithem** [comment]
add the home route path

File: src/app/home/home.component.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<body>
+
+    <div class="color"> 
+      <div class="image"> 
+        <img src="./assets/images/Screen_Shot_2018-04-11_at_2.18.31_PM_2048x.png" alt="Not Found" width="800" >
+        <div class="overlay"><p> if you're going to do it, do it safely<p></div>
```
Comment (zguesmi) [inline src/app/home/home.component.html]:
```suggestion
        <div class="overlay"><p> If you're going to do it, do it safely!<p></div>
```

File: src/app/home/home.component.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      </div>
+    </div>
+  
+    
+  
+      <div class="white">
+  
+      <h2>About Doit</h2>
+      <p>
+        &nbsp;&nbsp; Doit is a ToDo app that helps you organize your time and master the art of planning. Our app is simple to use, beautiful to see and more importantly secure to trust.
```
Comment (zguesmi) [inline src/app/home/home.component.html:16]: :+1:

**Review by zguesmi — CHANGES_REQUESTED** [review]

**Comment by zguesmi** [comment]
Good @HelaGdh! Can you just use clearer names for the image files.

**Comment by HelaGdh** [comment]
> Good @HelaGdh! Can you just use clearer names for assert files (images).

I wated to create a file hosting all the Images we're going to use in the project

**Comment by zguesmi** [comment]
Sorry I mean the image filenames not the name of the folder assets.

**Comment by HelaGdh** [comment]
homepage
hover over

**Review by zguesmi — APPROVED** [review]
:clap: :clap:

**Pull request merged**